projects/packages/videopress/src/client/block-editor/hooks/use-video-data/types.ts

```diff
@@ -7,7 +7,7 @@ import { VideoGUID, VideoId } from '../../blocks/video/types';
 export type UseVideoDataArgumentsProps = {
 	id?: VideoId;
 	guid?: VideoGUID;
-	isPrivate?: boolean;
```

Comment:
`isPrivate` was not in use anymore?

Reply:
nop. Let me change it in another PR. I would avoid running the tests again

Reply:
oh I see. I removed here 😅 Yes, `isPrivate` is not used now.